src/sentry/conf/server.py

```diff
@@ -118,6 +118,7 @@ def env(
 SENTRY_INTEGRATION_ERROR_LOG_REDIS_CLUSTER = "default"
 SENTRY_DEBUG_FILES_REDIS_CLUSTER = "default"
 SENTRY_MONITORS_REDIS_CLUSTER = "default"
+SENTRY_STATISTICAL_DETECTORS_REDIS_CLUSTER = "default"
```

Comment:
TODO: this needs to be broken out to a separate PR and updated in prod settings before it is used